funding/manager.go

```diff
@@ -4809,6 +4828,13 @@ func (f *Manager) handleErrorMsg(peer lnpeer.Peer, msg *lnwire.Error) {
 func (f *Manager) pruneZombieReservations() {
 	zombieReservations := make(pendingChannels)
 
+	reservationTimeout := f.cfg.ReservationTimeout
+	if f.cfg.Dev != nil {
```

Comment:
Also make sure there is an actual value in the dev config?
E.g. `if f.cfg.Dev != nil && f.cfg.Dev.ReservationTimeout != 0 {`?
Same with the sweeper interval below.

Reply:
Or use the `f.cfg.Dev.GetReservationTimeout()` method that always returns a non-zero value?

Reply:
Good idea, I will restructure to use yy's approach.